### ISSUE TITLE
WARNING: can't open config file

### ISSUE THREAD

**Issue by thom-nic** [riboseinc/homebrew-easy-rsa#7]
It seems easyrsa is expecting everything to be in the same directory as `easyrsa` which when installed via this formula is `/usr/local/bin`:
```
$ easyrsa make-ca
WARNING: can't open config file: /usr/local/bin/openssl-1.0.cnf

Easy-RSA error:

The OpenSSL config file cannot be found.
Expected location: /usr/local/bin/openssl-1.0.cnf
```

I installed via `brew tap riboseinc/easy-rsa && brew install easy-rsa`

**Comment by erikbor** [comment]
@thom-nic you need to export `EASYRSA_SSL_CONF` first.

```console
$ export EASYRSA_SSL_CONF="$(brew list -v easy-rsa | egrep openssl-.*.cnf)"
$ easyrsa init-pki

init-pki complete; you may now create a CA or requests.
Your newly created PKI dir is: /Users/test/cert/pki

$ easyrsa build-ca
Generating a 2048 bit RSA private key
...+++
...........................+++
writing new private key to '/Users/test/cert/pki/private/ca.key.rjanoivdDI'
Enter PEM pass phrase:
Verifying - Enter PEM pass phrase:
-----
You are about to be asked to enter information that will be incorporated
into your certificate request.
What you are about to enter is what is called a Distinguished Name or a DN.
There are quite a few fields but you can leave some blank
For some fields there will be a default value,
If you enter '.', the field will be left blank.
-----
Common Name (eg: your user, host, or server name) [Easy-RSA CA]:   

CA creation complete and you may now import and sign cert requests.
Your new CA certificate file for publishing is at:
/Users/test/cert/pki/ca.crt

$
```

Hope you find our Homebrew formulae useful!

**Comment by thom-nic** [comment]
Thanks!  I think it would be helpful to put something like 
```
NOTE: before using easyrsa you must set 

EASYRSA_SSL_CONF="$(brew --prefix easy-rsa)/share/openssl-1.0.cnf"
```
etc.  that prints during install. I can't find any documentation on printing out a notice during a homebrew install but I know other formula do it.

**Comment by ronaldtse** [comment]
@thom-nic this will actually be fixed using the approach in #9, so there should not be a need to run this command anymore. Thanks for reporting this issue and will ping you once it's done 👍
